react/components/ProductSummaryInlinePrice.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react'
+import { path } from 'ramda'
+import classNames from 'classnames'
+import { Link } from 'vtex.render-runtime'
+
+import AttachmentList from './AttachmentList'
+import ImageLoader from './ImageLoader'
+import ProductImage from './ProductImage'
+import ProductSummaryBuyButton from './ProductSummaryBuyButton'
+import ProductQuantityStepper from './ProductQuantityStepper'
+import ProductSummaryPrice from './ProductSummaryPrice'
+import ProductSummaryName from './ProductSummaryName'
+
+import productSummary from '../productSummary.css'
+
+class ProductSummaryInline extends Component {
```

Comment:
Why a class if you do not use its features?

Reply:
The other summary types should be changed too